api/src/test/resources/io/strimzi/api/kafka/model/KafkaAssembly.yaml

```diff
@@ -53,6 +53,13 @@ spec:
             }
         ]
       }
+    listeners:
+      tls:
+        authentication:
+          tlsClientAuthentication: required
+      plain: {}
+    authorization:
+      authorizer: SimpleAclAuthorizer
```

Comment:
Would presence of the `SimpleAclAuthorizer` cause deployment of the support for CRD->ACL operator stuff, or would that be configured separately? If separately what happens if it's configured but `authorizer != SimpleAclAuthorizer`?

Reply:
I think that depends how we implement it. If it is build into the USer Operator where most of our discussions were heading lately, it should probably automatically configure whether the Authorization part of the UO is enabled or not.